apps/train_loc/lib/train_loc/utilities/time.ex

```diff
@@ -48,16 +48,18 @@ defmodule TrainLoc.Utilities.Time do
 
   @spec get_service_date(DateTime.t()) :: Date.t()
   def get_service_date(current_time \\ DateTime.utc_now()) do
-    dt =
+    current_date =
       current_time
       |> in_local_tz
-      |> Timex.shift(hours: -3)
-      |> case do
-        %DateTime{} = dt -> dt
-        %Timex.AmbiguousDateTime{before: before} -> before
-      end
+      |> DateTime.to_date()
 
-    DateTime.to_date(dt)
+    local_time = in_local_tz(current_time)
```

Comment:
Suggestion: you're calling `in_local_tz` twice: can you get `current_date` from `local_time` instead?